docker/download_data.sh

```diff
@@ -0,0 +1,7 @@
+
```

Comment:
this syntax looks suspiciously bash to me, so...
```suggestion
#! /bin/bash
```